changelog.md

```diff
@@ -12,26 +12,31 @@ Thanks to our new contributors 👽🐷🐰🐯🐻!
 
 ## Core
 
-* fr: add navigation runner ([#11975](https://github.com/GoogleChrome/lighthouse/pull/11975))
 * handle timer throttling in DevTools to avoid timeouts ([#11987](https://github.com/GoogleChrome/lighthouse/pull/11987))
 * config: refactor config cloning for fraggle rock ([#11759](https://github.com/GoogleChrome/lighthouse/pull/11759))
 * console-messages: use source-location ([#11899](https://github.com/GoogleChrome/lighthouse/pull/11899))
 * errors-in-console: ignore BLOCKED_BY_CLIENT.Inspector errors ([#11901](https://github.com/GoogleChrome/lighthouse/pull/11901))
 * font-size: handle valueless attributes for inline styles ([#11934](https://github.com/GoogleChrome/lighthouse/pull/11934))
-* fr: add navigations to config ([#11957](https://github.com/GoogleChrome/lighthouse/pull/11957))
-* fr: add timespan runner ([#11944](https://github.com/GoogleChrome/lighthouse/pull/11944))
-* fr: filter configs by gather mode ([#11941](https://github.com/GoogleChrome/lighthouse/pull/11941))
 * full-page-screenshot: get the MAX_TEXTURE_SIZE from the browser ([#11847](https://github.com/GoogleChrome/lighthouse/pull/11847))
 * metrics: support FCP for all frames with devtools throttling ([#11874](https://github.com/GoogleChrome/lighthouse/pull/11874))
 * add type checking to page functions ([#11958](https://github.com/GoogleChrome/lighthouse/pull/11958))
```

Comment:
this is a misc

Reply:
> > add type checking ..
> 
> this is a misc

I don't think it's important enough to revert, it's a pretty minor changelog, but `misc` is really for stuff that we don't have a good category for and is ignorable enough by PRs that come later that it can just end up in the last bucket (`fix "fast" npm script` is a good example).

We don't really have a working definition of "Core" (beyond `"Driver, gather, (non-new) audits, LHR JSON, etc"`), but if we want to exclude type refactors from there, should we just make a new `types` category?